Sources/BraintreeDropIn/BTPaymentMethodNonce+DropIn.m

```diff
@@ -0,0 +1,31 @@
+#import "BTPaymentMethodNonce+DropIn.h"
+
+#ifdef COCOAPODS
+#import <Braintree/BTCardNonce.h>
+#import <Braintree/BTPayPalAccountNonce.h>
+#import <Braintree/BTVenmoAccountNonce.h>
+#import <Braintree/BTApplePayCardNonce.h>
+#else
+#import <BraintreeCard/BTCardNonce.h>
+#import <BraintreePayPal/BTPayPalAccountNonce.h>
+#import <BraintreeVenmo/BTVenmoAccountNonce.h>
+#import <BraintreeApplePay/BTApplePayCardNonce.h>
+#endif
+
+@implementation BTPaymentMethodNonce (DropIn)
+
+- (NSString *)paymentDescription {
```

Comment:
This is useful! I wonder if users of the base SDKs would want something like this or if this feels more Drop-in oriented.

Reply:
We used to have this in the base SDK called `localizedDescription`. We removed it in[ this PR](https://github.braintreeps.com/team-sdk/braintree-ios/pull/570). We removed it because it was essentially just redundant information that was available as other properties (ex: the paypal `email`, the venmo `username`).

Reply:
@demerino Do you think it is worth adding back into `braintree_ios` as a `paymentDescription` property? Susan and I went back and forth on it a bit.

Reply:
Thinking about this some more, I'm leaning towards adding this back into the base SDK with the updates we made here:
- Calling it `paymentDescription` (or `description`?) instead of `localizedDescription`
- For credit cards, returning `•••• ••11` instead of `ending in 11`

I think those two issues were a big part of why we wanted to remove `localizedDescription` in the first place, but fixing the issues feels better than removing the property altogether.

Reply:
Discussed this again with Susan over the phone - we're going to leave this property out of the base SDK now. We can consider adding it back in if we see inbounds for it.

Reasons for leaving it out:
- It doesn't apply to all payment methods, making it potentially _more_ confusing to merchants.
- ex: applePayNonce.paymentDescription is "Apple Pay" while the others are specific to the card/customer